netlogo-gui/src/main/sdm/gui/RateConnection.java

```diff
@@ -356,4 +359,31 @@ public boolean bivalentWrapper() {
     return rate.isBivalent();
   }
 
+  public void inputsC(Options<String> inList) {
+    rate.setSelected(inList.chosenName()); // not needed
+  }
+
+  public Options<String> inputsC() {
+    Options<String> inputs = new Options<String>();
+    inputs.addOption("Select", "null");
+    //String name = rate.getSelected(); // not needed for our case
+    inputs.selectByName("Select");
+
+    scala.collection.Iterator<String> ins = rate.getInputs().iterator();
+    while (ins.hasNext()) {
+      inputs.addOption(ins.next(), "A");
```

Comment:
Since we're not using the value parameter, I think we should either make it more meaningful, like `"unused"` instead of `"A"`, or alternatively add a comment explaining what it's for.